base/common/src/main/java/com/netscape/certsrv/request/RequestStatus.java

```diff
@@ -109,7 +127,7 @@
      */
     public static RequestStatus COMPLETE = new RequestStatus("complete");
 
-    private String label;
+    public String label;
```

Comment:
I think we can use `@JsonValue` for this class too.

Reply:
Ack